python/ray/util/sgd/tests/test_torch.py

```diff
@@ -334,11 +303,22 @@ def test_dataset(ray_start_4_cpus):
     model_creator = mlp_identity.model_creator
     optimizer_creator = mlp_identity.optimizer_creator
     dataset_creator = mlp_identity.dataset_creator
+
+    class DatasetOperator(TrainingOperator):
+        def setup(self, config):
+            model = model_creator(config)
+            optimizer = optimizer_creator(model, config)
+            loss = nn.MSELoss()
+
+            self.model, self.optimizer, self.criterion = self.register(
+                models=model,
+                optimizers=optimizer,
+                criterion=loss,
+                train_loader=None,
+                validation_loader=None)
```

Comment:
why does the `from_creator` not work here?

Reply:
It should work. I just didn't use it for this test.

Reply:
ok i just changed it to use `from_creator`.